api/v1alpha1/openapi.yaml

```diff
@@ -67,6 +72,12 @@ paths:
           description: The name of the repository to filter results by.
```

Comment:
Should the `repository` filter also be dropped in favour of `?fieldSelector=spec.repository=someRepo`?

BTW, If I attempt this at the moment, the error returned is error 500, with the message
```
unable to tokenize input using custom tokenizer: unknown field name "spec.repository"
```

Reply:
Hey, not all the fields were marked with selector. I marked only metadata and status. We can mark the spec at the model level to be a field that can be queried if required.

Reply:
Sure, I understand it's not a supported field yet. 

The comment was also about the fact that the error should be 400 instead of 500, as it's considered a bad request as it's referencing an unsupported filtering field. 

BTW, the same happens when specifying an unsupported sorting field in  "sortBy" query parameter and IMO it should also be error 400.
```
Error 500: unable to resolve field name "spec.template.spec.os"
```

Reply:
Please pull the latest changes and check now

Reply:
Fixed, thanks. For both filtering and sorting, API now returns error 400 for unknown fields.